### ISSUE TITLE
api.NavigationActivation - Safari TP supports it

### ISSUE THREAD

**Issue by saschanaz** [mdn/browser-compat-data#24963]
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

Support data for Safari TP

### What browsers does this problem apply to, if applicable?

Safari

### What did you expect to see?

It should be marked as supported in TP

### Did you test this? If so, how?

onpageswap = ev => console.log(ev.navigation) // gives NavigationActivation

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

https://github.com/mdn/browser-compat-data/pull/24410

### Do you have anything more you want to share?

Doesn't match with the data of `PageSwapEvent.navigation`

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/NavigationActivation

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `api.NavigationActivation`
* Report started: 2024-11-06T09:23:58.365Z

</details>

**Comment by caugner** [comment]
Initial WebKit impl: https://github.com/WebKit/WebKit/pull/30021

If the collector tests pass in Safari TP, we can set `version_added: "preview"`: https://mdn-bcd-collector.gooborg.com/tests/api/NavigationActivation

**Comment by saschanaz** [comment]
Perhaps we don't do that for TP though? Fixed by https://github.com/mdn/browser-compat-data/pull/28574 for 26.2.

**Comment by caugner** [comment]
> Perhaps we don't do that for TP though?

Yes, we don't have the capacity for updating statuses for preview versions proactively, so we also don't encourage contributors to open PRs for preview versions.
